Src/AwesomeAssertions/Equivalency/Selection/ExcludeMemberByNameSelectionRule.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using AwesomeAssertions.Equivalency;
+
+namespace AwesomeAssertions.Equivalency.Selection;
+
+/// <summary>
+/// Selection rule that removes a particular member from the structural comparison if its name matches the specified criteria.
+/// </summary>
+internal class ExcludeMemberByNameSelectionRule : IMemberSelectionRule
```

Comment:
BTW, I think we could benefit from having an abstract `MemberSelectionRule` base implementation.

Reply:
BTW @IT-VBFK and @cbersch, I think we could benefit from having an abstract `MemberSelectionRule` base implementation.